core-api/src/main/java/com/sequenceiq/cloudbreak/client/PkiUtil.java

```diff
@@ -167,6 +179,14 @@ public static String convert(X509Certificate cert) {
         }
     }
 
+    public static String convert(PemObject pemObject) {
```

Comment:
These are added so that they are used from dps-provisioning. Source code is at https://github.com/hortonworks/dps. Please see the jira for more context.